src/Shared/E2ETesting/SeleniumStandaloneServer.cs

```diff
@@ -112,7 +112,7 @@ private static async Task InitializeInstance(ITestOutputHelper output)
             {
                 // Just create a random tracking folder on helix
                 trackingFolder = Path.Combine(Directory.GetCurrentDirectory(), Path.GetRandomFileName());
-                Directory.Create(trackingFolder));
+                Directory.CreateDirectory(trackingFolder));
```

Comment:
What's the difference between these two?

Reply:
One actually exists :)